### PR TITLE
Gh 2343/intro screen after generating key

### PR DESCRIPTION
**Pull request by MouazAlzahabi**
Handles #2343 

 screenshot
Simulator Screen Shot - iPhone 13 Pro - 2022-06-10 at 18 01 22

Design
Screen Shot 2022-06-10 at 6 03 49 PM

### CODE REVIEW

**Comment by DmitryBespalov** [comment]
@MouazAlzahabi can you please attach a 'screenshot' vs 'design' image to the pull-request?

**Comment by DmitryBespalov** [comment]
- The title's font style is not as it looks in figma
- The "not this time" button is too close to the bottom of the page

File: Multisig/UI/App/ViewControllerFactory.swift [hunk 1]
```diff
@@ -84,9 +84,23 @@ enum ViewControllerFactory {
             UIBarButtonItem(barButtonSystemItem: .close, target: vc, action: #selector(CloseModal.closeModal))
     }
 
+    static func removeNavigationItem(_ vc: UIViewController) {
```
Comment (DmitryBespalov) [inline Multisig/UI/App/ViewControllerFactory.swift:87]: 👍

File: Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/CreateInviteOwnerIntroViewController.swift [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        notThisTimeButton.setText("Not this time", .plain)
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        Tracker.trackEvent(.addAsOwnerIntro)
+    }
+
+    // Called when user swipes down the modal screen
+    func presentationControllerDidDismiss(_ presentationController: UIPresentationController) {
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/CreateInviteOwnerIntroViewController.swift]: can you check that this actually called? I think the swipe-to-dismiss is handled in another place (in navigation controller), but not sure.

When I tried to swipe down but then moved the finger up, the screen still swiped down, however I expected it to stay.

File: Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/GenerateKeyFlow.swift [hunk 3]
```diff
@@ -97,6 +109,14 @@ class GenerateKeyFlow: AddKeyFlow {
         push(flow: replaceOwnerFlow)
     }
 
+    func didInviteToAddKey(openKeyDetails: Bool = true) {
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/GenerateKeyFlow.swift]: I think this method is not needed, you could use the `didAddKeyAsOwner`. What is the reason that this duplicate method exists?

File: Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/CreateInviteOwnerIntroViewController.swift (hunk 2 above)
Comment (MouazAlzahabi) [reply Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/CreateInviteOwnerIntroViewController.swift]: It turns out that it's general issue that we have unfortunately
Let's fix it separately

**Review by DmitryBespalov — APPROVED** [review]

**Pull request merged**